bindings/node.js/kzg.cxx

```diff
@@ -260,17 +237,19 @@ Napi::Value VerifyKzgProof(const Napi::CallbackInfo& info) {
 
 // verifyBlobKzgProof: (blob: Blob, commitmentBytes: Bytes48, proofBytes: Bytes48, setupHandle: SetupHandle) => boolean;
```

Comment:
Can you please add some comments on the expected return values (and exceptions) of this function, and all other verification functions (they are the most important functions here so let's take good care of them).

Reply:
added.  I also put the comments in the JS file so they show up with intellisense.